Added README to repo and inv_kinematics, Added gitignore file

jkkrupinski opened the pull request:
In reference to https://github.com/KNR-PW/MelsonKNR/issues/1

jkkrupinski committed:
Added README to repo and inv_kinematics, Added gitignore file

pktiuk commented on README.md:
```diff
@@ -0,0 +1,22 @@
+# This repository contains the catkin workspace for KNR Melson humanoid robot
+
+
+
+
+
```
Trochę mało tego opisu    
Skoro to główne repo to warto się nieco rozpisać o samym projekcie melson.

Można też dać kilka linków, np do waszej podstrony na stronie koła: https://knr.edu.pl/melson/   
Na samym dole można też dopisać linki do innych repozytoriów melsonowych, które są jakoś powiązane.

BartlomiejK2 committed:
Update README.md  for inverse_kinematics

Dodałem dokładniejszy opis Pythonowych funkcji/skryptów które są używane do generowania trajektorii i odwrotnej kinematyki.

BartlomiejK2 committed:
Update README.md
